tests/test_md.py

```diff
@@ -70,6 +71,23 @@ def test_NVT_const_T(cu_slab):
     assert all([at.info['MD_temperature_K'] == 500.0 for at in atoms_traj])
 
 
+def test_NVT_Langevin_const_T(cu_slab):
+
+    calc = EMT()
+
+    inputs = ConfigSet(cu_slab)
+    outputs = OutputSpec()
+
+    atoms_traj = md.md(inputs, outputs, calculator=calc, integrator="Langevin", steps=300, dt=1.0,
+                           temperature = 500.0, temperature_tau=100/fs)
```

Comment:
why are you dividing by fs here?

Reply:
I wanted set friction 0.01. since it's multiplied with fs in `md/__init.py`, I divided here. 
```
174                 elif integrator == "Langevin":
175                     md_constructor = Langevin
176                     stage_kwargs["friction"] = 1 / (temperature_tau * fs)
```